src/main/resources/templates/typescript-language-server/clientSettings.json

```diff
@@ -9,7 +13,6 @@
       "clientSide": {
         "formatOnCloseBrace": true,
         "formatOnStatementTerminator": true,
-        "formatOnStatementTerminatorCharacters": ";",
```

Comment:
This could be removed since we're setting the language grammar-level statement terminator now.